evergreen.yml

```diff
@@ -247,6 +247,21 @@ functions:
           # we only run asan on the unit tests for now
           cargo test --target x86_64-unknown-linux-gnu unit
 
+  "run result set test":
```

Comment:
Since the data loader didn't have any test tasks to run, I left evergreen changes out of #15 . Can you update the relevant evergreen task(s) to include `cargo run data_loader` to run the data loader after ADF starts?

Reply:
Added, thanks for the suggestion.  Also, it looks like the run command has a --bin option: `cargo run --bin data_loader`